ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__techpreview-periodic-4.16.yaml

```
@@ -0,0 +1,237 @@
+    workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn-etcd-scaling
+  minimum_interval: 168h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    observers:
+      enable:
+      - observers-resource-watch
+    workflow: openshift-e2e-openstack-etcd-scaling
+- as: e2e-openstack-dualstack
+  minimum_interval: 96h
+  steps:
+    cluster_profile: openstack-hwoffload
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    workflow: openshift-e2e-openstack-dualstack
+- as: e2e-openstack-dualstack-v6primary
+  minimum_interval: 96h
+  steps:
+    cluster_profile: openstack-hwoffload
+    env:
+      CONFIG_TYPE: dualstack-v6primary
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_SKIPS: when FIPS is disabled the HAProxy router should serve routes when
+        configured with a 1024-bit RSA key\|should redirect on blob pull
+    workflow: openshift-e2e-openstack-dualstack
+- as: e2e-openstack-ccpmso
+  minimum_interval: 240h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_SUITE: experimental/reliability/minimal
+    test:
+    - as: openstack-test-cpms
+      commands: |
+        dir=$GOPATH/src/github.com/openshift/cluster-control-plane-machine-set-operator
+        git clone https://github.com/openshift/cluster-control-plane-machine-set-operator --branch release-4.16 --depth=1 $dir
+        export OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE="$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")"
+        cd $dir
+        make e2e-periodic
+      from: stolostron
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 3h0m0s
+    - ref: openshift-e2e-test
+    - ref: openstack-test-openstack
+    workflow: openshift-e2e-openstack-ipi
+  timeout: 7h0m0s
+- as: e2e-openstack-ccpmso-zone
+  minimum_interval: 240h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    env:
+      ETCD_ON_LOCAL_DISK: "true"
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_SUITE: experimental/reliability/minimal
+      USE_RAMFS: "false"
+    test:
+    - as: openstack-test-cpms
+      commands: |
+        dir=$GOPATH/src/github.com/openshift/cluster-control-plane-machine-set-operator
+        git clone https://github.com/openshift/cluster-control-plane-machine-set-operator --branch release-4.16 --depth=1 $dir
+        export OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE="$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")"
+        cd $dir
+        test -f "${SHARED_DIR}/proxy-conf.sh" && source "${SHARED_DIR}/proxy-conf.sh"
+        make e2e-periodic
+      from: stolostron
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 3h0m0s
+    - ref: openshift-e2e-test
+    - ref: openstack-test-openstack
+    workflow: openshift-e2e-openstack-ipi
+  timeout: 7h0m0s
+zz_generated_metadata:
+  branch: main
+  org: shiftstack
+  repo: shiftstack-ci
+  variant: techpreview-periodic-4.16
```

```suggestion
  variant: techpreview
```

Looking at the job names from the rehearsal, e.g.:

```
periodic-ci-shiftstack-shiftstack-ci-main-techpreview-periodic-4.16-e2e-openstack-proxy 	N/A 	periodic 	Periodic changed
```
 `techpreview` stutters (i.e. it is repeated twice) and `4.16` seems like it might also be redundant (this is for `main`) or not future proof...